cmake/FindLibva.cmake

```diff
@@ -23,7 +23,7 @@
 
 find_library(LIBVA_LIBRARY NAMES va HINTS /opt/amdgpu/lib/x86_64-linux-gnu /opt/amdgpu/lib64 /usr/lib/x86_64-linux-gnu /usr/lib64)
 find_library(LIBVA_DRM_LIBRARY NAMES va-drm HINTS /opt/amdgpu/lib/x86_64-linux-gnu /opt/amdgpu/lib64 /usr/lib/x86_64-linux-gnu /usr/lib64)
-find_path(LIBVA_INCLUDE_DIR NAMES va/va.h PATHS /opt/amdgpu/include NO_DEFAULT_PATH)
+find_path(LIBVA_INCLUDE_DIR NAMES va/va.h PATHS /opt/amdgpu/include /usr/include NO_DEFAULT_PATH)
```

Comment:
rocDecode requires a minimum VA-API version of 1.16, which is necessary for AMD Mesa. We have intentionally disabled the system libva path and have specifically utilized the libva-amdgpu-dev package to build rocDecode.

Reply:
It still prefers the libva installed in /opt/amdgpu, and the >= 1.16 version check is there and will fail for old versions.

Reply:
There is no need to add a system path for libva, as we specifically use libva-amdgp-dev for the build process.